### PR TITLE
[PULP-206] Refactor nightly

### PR DESCRIPTION
**Pull request by mdellweg**
The first commit is the actual fix.
The second commit is to make sure it does not happen again.

### CODE REVIEW

File: templates/github/.github/workflows/nightly.yml.j2 [hunk 1]
```diff
@@ -109,20 +112,20 @@ jobs:
...
       {{ run_script(name="Install", file="install.sh") | indent(6) }}
 
       {{ run_script(name="Before Script", file="before_script.sh", extra_env={"REDIS_DISABLED": "${{ contains('%s', matrix.env.TEST) }}" % (disabled_redis_runners | default(["TILT"]) | join(' '))}) | indent(6) }}
 
-      {{ run_script(name="Performance Test", file="script.sh") | indent(6) }}
+      {{ run_script(name="Script", file="script.sh") | indent(6) }}
 
       {{ display_logs() | indent(6) }}
 {%- endif %}
+...
```
Comment (pedro-psb) [inline templates/github/.github/workflows/nightly.yml.j2:63]: Why this `...` ?

Comment (mdellweg) [reply templates/github/.github/workflows/nightly.yml.j2:63]: This is being polite in yaml.

Comment (mdellweg) [reply templates/github/.github/workflows/nightly.yml.j2:63]: I'm also on a crusade quote all string values.

Comment (pedro-psb) [reply templates/github/.github/workflows/nightly.yml.j2:63]: I'm following the quote crusade, but i was not aware of the ... thing.

**Review by pedro-psb — APPROVED** [review]
LGTM.

I see the refactor removed a duplication of `install_python_deps` which is good!

File: templates/github/.github/workflows/nightly.yml.j2 (hunk 1 above)
Comment (pedro-psb) [reply templates/github/.github/workflows/nightly.yml.j2:63]: Oh yeah, its usually used in multifile yaml. I remembered it as was just `---`.

**Pull request merged**